lib/Net/Async/Github.pm

```diff
@@ -211,6 +211,39 @@ sub pr {
     )
 }
 
+=head2 prs
+
+Returns information of all PRs of given repository.
+
+Expects the following named parameters:
+
+=over 4
+
+=item * owner - which user or organisation owns this PR
+
+=item * repo - which repo it's for
+
+=back
+
+Return Ryu::Source object that resolves to the object of all PRs.
```

Comment:
not sure what  does this mean

`resolves to the object of all PRs.`

Reply:
copied from other comments.